tests/DCPS/DynamicData/publisher.cpp

```diff
@@ -2,17 +2,21 @@
 
 #include <tests/Utils/DistributedConditionSet.h>
 
-#include <dds/DCPS/Service_Participant.h>
+#include <dds/DCPS/DCPS_Utils.h>
 #include <dds/DCPS/Marked_Default_Qos.h>
+#include <dds/DCPS/Service_Participant.h>
 #include <dds/DCPS/StaticIncludes.h>
-#include <dds/DCPS/DCPS_Utils.h>
+
 #include <dds/DCPS/XTypes/DynamicTypeSupport.h>
 #include <dds/DCPS/XTypes/DynamicDataFactory.h>
+
```

Comment:
```suggestion
#include <dds/DCPS/XTypes/DynamicTypeSupport.h>
#include <dds/DCPS/XTypes/DynamicDataFactory.h>
```

There shouldn't be any separation between these because they're all part of the OpenDDS core libraries.